<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="firstelement">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="secondelement">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>2.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="favourite1">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="favourite2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>540</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="ProductTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>281</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="FavouriteTable">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>281</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="ComparisonTable1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="ComparisonTable2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>371</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
